queue_services/entity-emailer/src/entity_emailer/message_tracker/tracker.py

```diff
@@ -106,7 +106,8 @@ def get_message_context_properties(queue_msg: nats.aio.client.Msg):
             return create_message_context_properties(etype, message_id, None, None, False)
 
         if etype in ('dissolution', 'registration', 'changeOfRegistration',
-                     'restoration', 'specialResolution', 'correction', 'amalgamationApplication', 'continuationIn') \
+                     'restoration', 'specialResolution', 'correction', 'amalgamationApplication', 'continuationIn',
+                     'ceaseReceiver') \
```

Comment:
Does ceaseReceiver require email for both status (PAID and COMPLETED), If only one email then move ceaseReceiver to the above if (line 100)

Reply:
This is not clear yet. I will check this again and change accordingly

Reply:
Updated this. I will change if needed for PAID as well after I get confirmation.